app/models/bot/alegre.rb

```diff
@@ -493,6 +493,15 @@ def self.add_relationship(pm, pm_id_scores, parent_id)
     end
   end
 
+  def self.get_indexing_model(pm)
+    type = self.get_pm_type(pm)
+    if type == text
```

Comment:
also I think text should be a string 'text' right?

Reply:
Good catch!